portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -36,9 +36,25 @@
   @Override
   public void init() {
       messages = new ArrayList<String>();
```

Comment:
Since you know messages is already a `List<String>`, you can let it infer: `new ArrayList<>();`